api_app/api/routes/airlock_resource_helpers.py

```diff
@@ -42,12 +50,28 @@ async def save_and_publish_event_airlock_request(airlock_request: AirlockRequest
         raise HTTPException(status_code=status.HTTP_503_SERVICE_UNAVAILABLE, detail=strings.EVENT_GRID_GENERAL_ERROR_MESSAGE)
 
 
-async def update_and_publish_event_airlock_request(airlock_request: AirlockRequest, airlock_request_repo: AirlockRequestRepository, user: User, new_status: AirlockRequestStatus, workspace: Workspace, request_files: List[AirlockFile] = None, status_message: str = None, airlock_review: AirlockReview = None):
+async def update_and_publish_event_airlock_request(
+        airlock_request: AirlockRequest,
+        airlock_request_repo: AirlockRequestRepository,
+        user: User,
+        workspace: Workspace,
+        new_status: AirlockRequestStatus = None,
```

Comment:
new_status can now be None to account for the case when the status isn't changed